.github/file-filters.yml

```diff
@@ -35,6 +35,7 @@ frontend_components_modified_lintable:
 backend_lintable: &backend_lintable
   - '**/*.py'
   - 'requirements-base.txt'
+  - 'requirements-dev-only-frozen.txt'
```

Comment:
Why is the word `only` in the file name? I'm trying to determine what it's trying to imply.

Reply:
I think it means that it only contains the dev requirements and not the base requirements -- we'll eventually have 3 flavors:
- base only
- dev only
- base + dev

Reply:
dev only is just a fast path for certain CI jobs that don't need sentry installed like linting, test-tools.